igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -53,7 +53,7 @@ class TravisService implements BuildService {
     private Accounts accounts
 
     TravisService(String travisHostId, String baseUrl, String githubToken, TravisClient travisClient, TravisCache travisCache) {
-        this.groupKey     = "travis-${travisHostId}"
+        this.groupKey     = "${travisHostId}"
```

Comment:
I would recommend keep the prefix here, and then removing it [here](https://github.com/spinnaker/igor/blob/master/igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/TravisConfig.groovy#L63), so the key names are generated consistently. Are the keys every used in caching? If so, then it's especially important so they don't clash with jenkins keys.